mattermost-plugin/server/plugin.go

```diff
@@ -90,7 +90,7 @@ func (p *Plugin) OnActivate() error {
 		DBTablePrefix:           "focalboard_",
 		UseSSL:                  false,
 		SecureCookie:            true,
-		WebPath:                 "./plugins/focalboard/pack",
```

Comment:
should we use `path.Join` here?

Reply:
I felt uneasy about it as our default value for the plugins directory is `./plugins` and putting it through `path.Join("./plugins", "focalboard/pack")` yields `plugins/focalboard/pack` without the `./`. In any case, I'll test it out and send another PR if it works fine with relative paths

Reply:
That and the fact that we do the same at the `ServerRoot` just seven lines above :P